boards/arm/mimxrt1060_evk/mimxrt1060_evk-pinctrl.dtsi

```diff
@@ -2,57 +2,66 @@
  * Copyright (c) 2022, NXP
  * SPDX-License-Identifier: Apache-2.0
  *
- * Generated by rt_cfg_utils.py on 2022-01-27
+ * Generated by rt_cfg_utils.py on 2022-02-04
  */
 
 #include <nxp/nxp_imx/rt/mimxrt1062-iomuxc.h>
 
 &pinctrl {
 	lpuart1_default: lpuart1_default {
-		iomuxc_gpio_ad_b0_12_lpuart1_tx: iomuxc_gpio_ad_b0_12_lpuart1_tx {
-			pinmux = <IOMUXC_GPIO_AD_B0_12_LPUART1_TX>;
-			bias-bus-hold;
-			input-enable;
-		};
-
-		iomuxc_gpio_ad_b0_13_lpuart1_rx: iomuxc_gpio_ad_b0_13_lpuart1_rx {
-			pinmux = <IOMUXC_GPIO_AD_B0_13_LPUART1_RX>;
+		group0 {
+			mux = <IOMUXC_GPIO_AD_B0_12_LPUART1_TX_MUX>,
+				<IOMUXC_GPIO_AD_B0_13_LPUART1_RX_MUX>;
+			mode = <IOMUXC_GPIO_AD_B0_12_LPUART1_TX_MODE>,
+				<IOMUXC_GPIO_AD_B0_13_LPUART1_RX_MODE>;
+			input = <IOMUXC_GPIO_AD_B0_12_LPUART1_TX_INPUT>,
+				<IOMUXC_GPIO_AD_B0_13_LPUART1_RX_INPUT>;
+			daisy = <IOMUXC_GPIO_AD_B0_12_LPUART1_TX_DAISY>,
+				<IOMUXC_GPIO_AD_B0_13_LPUART1_RX_DAISY>;
+			cfg = <IOMUXC_GPIO_AD_B0_12_LPUART1_TX_CFG>,
+				<IOMUXC_GPIO_AD_B0_13_LPUART1_RX_CFG>;
```

Comment:
this is definitely a solution to the 32-bit limitation. What you mention of using nodes containing all these fields in a single place is also a good solution (less verbose, so probably worth giving it a try).